x/emissions/keeper/queryserver/query_server_losses_test.go

```diff
@@ -207,3 +208,193 @@ func (s *QueryServerTestSuite) TestGetDeleteDelegateStake() {
 	s.Require().Equal(removalInfo.Delegator, retrievedInfo.Delegator)
 	s.Require().Equal(removalInfo.Amount, retrievedInfo.Amount)
 }
+
+func (s *QueryServerTestSuite) TestGetOpenReputerSubmissionWindows() {
```

Comment:
i would write this as a table-driven test.  would make it easy to add more cases.  i can think of maybe 4-6 that would be useful -- and less code to do it this way.

it actually helps w/r/t clarity to do so.  it's a more compact and readable definition of expectations.  and can be statically analyzed more easily if we ever need to build more tools like that (we've got a head start on it via our custom linters)